WcaOnRails/app/models/team.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 class Team < ApplicationRecord
+  BOARD_FRIENDLY_ID = 'board'
```

Comment:
On further thought, what if we changed all these to something like:

```ruby
def self.board
  Team.find_by_friendly_id!('board')
end
```

Then, everywhere else, we can simply refer to `Team.board`, which I think would clean up a lot of the `Team.find_by_friendly_id(Team::*_FRIENDLY_ID)` statements all over this PR.

If we *do* ever want a team's id or friendly id, we always can do `Team.board.id` or `Team.board.friendly_id`.

What do you think? Good idea, or just crazy? @jonatanklosko and @viroulep in case they have some thoughts.

Reply:
I think that's reasonable. I don't know from a sytax point of view, but from a practical one it sounds nice.